### DISCUSSION TITLE
searx appearance

### DISCUSSION THREAD

**Discussion by diluex** [searx/searx#721] [Installation issue: custom]
if deploy searx locally by `python ./webapp.py` its appearance is exactly like on searx.me.
running as a daemon deployed by uwsgi, it's just black/white text and also arranged differently. I've followed the instructions on https://asciimoo.github.io/searx/dev/install/installation.html.
what's gone wrong?

[searx.pdf](https://github.com/asciimoo/searx/files/513230/searx.pdf)

[searx.ini]

[uwsgi]
# Who will run the code

uid = searx
gid = searx
# disable logging for privacy

disable-logging = true
# Number of workers (usually CPU count)

workers = 4
# The right granted on the created socket

chmod-socket = 666
# Plugin to use and interpretor config

single-interpreter = true
master = true
plugin = python
# Module to import

module = webapp
# Plugin to use and interpretor config

single-interpreter = true
master = true
plugin = python
# Module to import

module = webapp
# Virtualenv and python path

virtualenv = /usr/local/searx/searx/searx-ve/
pythonpath = /usr/local/searx/searx/
chdir = /usr/local/searx/

**Comment by devonhk** [comment]
So it looks like the path to your static files isn't setup right (css, js, img). If you open the network tab in your browser you can confirm this if you see a bunch of 404s.

Also have you tried deploying with docker? it might be easier
